install.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -e
+
+install_pipelines(){
+
+    echo "Checking dependencies..."
+    if [ ! -n "$(which python)" ]; then
+        echo "Need python installed !";
+        exit 1;
+    fi
+    if [ ! -n "$(which pip)" ]; then
+        echo "Need pip installed !";
+        exit 1;
+    fi
+    if [ ! -n "$(which git)" ]; then
+        echo "Need git installed !"
+        exit 1;
+    fi
+
+    echo "Installing pipelines..."
```

Comment:
I'd add a step to ask whether the user wants the latest dev or the mainstream version

```
echo -n "Which version of pipelines to install? (dev/LATEST) "
read version
if [ -z "$version" ]; then
    version=LATEST
fi

if [ "$version" == "dev" ]; then
    sudo pip install git+git://github.com/Wiredcraft/pipelines@dev > /dev/null
elif [ "$version" == "LATEST" -o "$version" == "latest" ]; then
    sudo pip install pipelines
else
    echo "Invalid version."
    exit 1
fi
```